doc/source/train/examples/lightning/lightning_mnist_example.ipynb

```diff
@@ -154,11 +145,13 @@
     "        acc = self.accuracy(logits, y)\n",
     "        return loss, acc\n",
     "\n",
-    "    def validation_epoch_end(self, outputs):\n",
-    "        avg_loss = torch.stack([x[\"val_loss\"] for x in outputs]).mean()\n",
-    "        avg_acc = torch.stack([x[\"val_accuracy\"] for x in outputs]).mean()\n",
+    "    def on_validation_epoch_end(self):\n",
+    "        avg_loss = torch.stack(self.eval_loss).mean()\n",
```

Comment:
nit: Can you explain in the PR description why these changes are needed (PTL 2.0) and whether these changes are backwards compatible with older versions?

Reply:
Done